zipline/testing/fixtures.py

```diff
@@ -1688,12 +1693,16 @@ def make_adjustment_db_conn_str(cls):
     def init_class_fixtures(cls):
         super(WithAdjustmentReader, cls).init_class_fixtures()
         conn = sqlite3.connect(cls.make_adjustment_db_conn_str())
-        cls.make_adjustment_writer(conn).write(
-            splits=cls.make_splits_data(),
-            mergers=cls.make_mergers_data(),
-            dividends=cls.make_dividends_data(),
-            stock_dividends=cls.make_stock_dividends_data(),
-        )
+        # Silence numpy DeprecationWarnings which cause nosetest to fail
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore", DeprecationWarning)
```

Comment:
Looks like this is from using [system-dependent `np.integer`](https://github.com/quantopian/zipline/blob/e63871f7e5affb0431a9d4d3780f14d15a315aba/zipline/data/adjustments.py#L50) instead of `int64` or `int32`. My initial thought is we should be stricter, since we control the data format. I say we leave what you have here and open an issue.